### PR TITLE
Add notifications in views for member add/edit/remove

### PR DESCRIPTION
**Pull request by aru31**

### CODE REVIEW

File: utils/get_category.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+
+from categories.models import Category
+
+
+def get_category():
+    """
```
Comment (dhruvkb) [inline utils/get_category.py:7]: Documentation is missing. This does not qualify as documentation.

File: utils/get_category.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    :return: category
+    """
+
+    service_name = settings.DISCOVERY.get_app_configuration('groups').nomenclature.name
```
Comment (dhruvkb) [inline utils/get_category.py]: Is this within 80 characters?

File: views/membership.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+from rest_framework import status
+from rest_framework.response import Response
```
Comment (dhruvkb) [inline views/membership.py]: Import `response` from DRF and use `response.Response` wherever. Makes imports much cleaner.

**Review by dhruvkb — CHANGES_REQUESTED** [review]

**Review by dhruvkb — CHANGES_REQUESTED** [review]
https://github.com/IMGIITRoorkee/omniport-service-developer/pull/1#pullrequestreview-338421936 applies here.

File: utils/get_category.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    get or create category instance with slug
+    :return: category object
+    """
+
+    service_name = settings.DISCOVERY.get_app_configuration(
+        'groups'
+    ).nomenclature.name
+    category, _ = Category.objects.get_or_create(
+        slug=service_name,
+        name=service_name
```
Comment (ayushjn20) [inline utils/get_category.py]: Shouldn't the category node's name be the `verbose_name` of `groups` service?

**Comment by dhruvkb** [comment]
Use the same feedback as given in the other PR IMGIITRoorkee/omniport-service-developer#1.

File: utils/get_category.py (hunk 4 above)
Comment (ayushjn20) [reply utils/get_category.py]: @aru31 Any progress?

**Review by ayushjn20 — APPROVED** [review]

File: utils/get_category.py (hunk 4 above)
Comment (aru31) [reply utils/get_category.py]: Done

**Pull request merged**